libminifi/src/core/yaml/YamlConfiguration.cpp

```diff
@@ -656,6 +656,17 @@ void YamlConfiguration::parseConnectionYaml(YAML::Node *connectionsNode, core::P
         }
         connection->setDestinationUUID(destUUID);
 
+        // Flow File Expiration Duration in MilliSeconds
+        uint64_t expirationDuration = 0;
```

Comment:
This can be scoped inside the "if" block

Reply:
Indeed, thanks!